models/migrations/v229.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"xorm.io/xorm"
+)
+
+func addIndexForHookTask(x *xorm.Engine) error {
+	type HookTask struct {
+		ID     int64  `xorm:"pk autoincr"`
+		HookID int64  `xorm:"index"`
+		UUID   string `xorm:"unique"`
+	}
+
+	return x.Sync2(new(HookTask))
```

Comment:
The comment said

```
// Sync2 synchronize structs to database tables
// Depricated
```


Why still using it?

Reply:
Done.